### ISSUE TITLE
Is it possible to have the actions done as a bot?

### ISSUE THREAD

**Issue by louim** [jonabc/licensed-ci#212]
Hey! Thanks for creating this action.

I was wondering if it's possible to have the actions (like the commits, branch and comments creation) done as non user? Is there a different token I could use for this?I'm currently using a PAT. I'm trying not to get pinged each time a branch/issue is created 😄

**Comment by jonabc** [comment]
@louim there is no default token available specifically for this action.  That said, you should be able to use any GitHub authorization token that's valid for the actions that you mentioned.

Also, you don't necessarily need to use a custom PAT, `${{ secrets.GITHUB_TOKEN }}` may work for your needs.  There are [scenarios where the default actions token may not work]https://github.com/jonabc/licensed-ci#authentication), and it looks like I need to update the readme to also call out [a limitation from the actions help documentation](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow).  If none of those apply to your usage though, you might be ok with `GITHUB_TOKEN`?

> When you use the repository's GITHUB_TOKEN to perform tasks, events triggered by the GITHUB_TOKEN, with the exception of workflow_dispatch and repository_dispatch, will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events occur.

**Comment by louim** [comment]
@jonabc yeah, sorry I wasn't clear, I'm using a custom PAT to get around the reason you mention (default action token not re-triggering a run).

**Comment by jonabc** [comment]
Ah I see.  Yeah unfortunately there's no common PAT or other authorization mechanism I'm aware of to bypass notifications when using a PAT on your account. I haven't kept close track of changes or updates to notifications and token usage though, maybe there's something in the help docs?

If not, would creating a helper/bot type user account be an option and using PATs created on the other account to restrict or manage notifications?

**Comment by gr2m** [comment]
you can use https://github.com/actions/create-github-app-token. Just register an app that you want to be the actor with the permissions and access needed, then pass the token into this action and you should be golden
